File: Data Structures & Algorithms/remove-element/submission-0.cs
public class Solution {
    public int RemoveElement(int[] nums, int val) {
        int currentInsertPos = 0;
        int count = 0;

        for (int i = 0; i < nums.Length; i++) {
            if (nums[i] == val) {
                count++;
            }
            else {
                nums[currentInsertPos] = nums[i];
                currentInsertPos++;
            }
        }

        return nums.Length - count;
    }
}

File: Data Structures & Algorithms/majority-element-ii/submission-0.py
from collections import Counter

class Solution:
    def majorityElement(self, nums: List[int]) -> List[int]:
        counts = Counter(nums)
        result = []

        for key, value in counts.items():
            if value > len(nums) / 3:
                result.append(key)
        
        return result
